Essentials/src/config.yml

```diff
@@ -612,6 +612,15 @@ minimum-pay-amount: 0.001
 # For 1'234,50 use fr-ch
 #currency-symbol-format-locale: en-US
 
+# Whether to enable modern, UUID-friendly support for Vault or not.
+# If you are having issues with plugins utilising EssentialsX's economy, you may want to disable this.
+use-new-vault-hook: true
+
+# Whether to disable Vault's legacy EssentialsEco hook or not.
+# This may be necessary to avoid conflicts with EssentialsX.
+# This will only work if use-new-vault-hook is enabled!
```

Comment:
I'm curious as to what would happen if both were set to `false`. Would you disable the plugin and print out an error then? Or will you simply override what is configured and load the new vault hook?

Reply:
If both are set to false, this does nothing - Vault's built in Essentials hook will be used instead.

Reply:
I see.